### ISSUE TITLE
Full license text in JSON report

### ISSUE THREAD

**Issue by sebastianludwig** [jaredsburrows/gradle-license-plugin#222]
Currently only the HTML report contains the full license text. We'd love to use your plugin but we need the full license text in the JSON report. Would you be open to a PR?

**Comment by jaredsburrows** [comment]
Yes. I am always willing to accept PRs!. Can we add this new functionality while keeping the old functionality?

**Comment by sebastianludwig** [comment]
Hey, sure, we can just add a new config value. A slight problem is that the defaults currently vary between JSON (not included) and HTML (included). Would it be okay with you to change the default of either one of them? Then we could have a config value like `includeFullLicenseTexts = true/false` which does the same for all reports. If yes, what should be the default value? `true`?

**Comment by jaredsburrows** [comment]
Yeah. Should be `true`.

**Comment by jaredsburrows** [comment]
@sebastianludwig Do you mind sending a PR for this?

**Comment by sebastianludwig** [comment]
Hey, I moved projects and am currently not working with Android. If that changes in the future, I'll happily come back to my offer. Sorry for not getting back to you any earlier and for not being able to help :-/
